src/credentials/tests/TestFabricTable.cpp

```diff
@@ -163,29 +180,38 @@ void TestUpdateLastKnownGoodTime(nlTestSuite * inSuite, void * inContext)
             // after installing the new fabric.
             NL_TEST_ASSERT(inSuite, fabricTable.GetLastKnownGoodChipEpochTime(lastKnownGoodTime) == CHIP_NO_ERROR);
             NL_TEST_ASSERT(inSuite, lastKnownGoodTime == buildTime);
-        }
-        {
-            // Test reloading last known good time from persistence.
-            FabricTable fabricTable;
-            NL_TEST_ASSERT(inSuite, fabricTable.Init(&testStorage) == CHIP_NO_ERROR);
 
-            // Verify that last known good time was retained.
-            System::Clock::Seconds32 lastKnownGoodTime;
+            // Verify that calling the fail-safe roll back interface does change
+            // last known good time, as it hadn't been updated in the first place.
```

Comment:
I don't understand this comment.  We are not checking that the time has changed, as far as I can tell.

Reply:
Fixed